sys/kern/interrupt.c

```diff
@@ -150,7 +166,7 @@ static void intr_thread(void *arg) {
     intr_event_t *ie = ih->ih_event;
 
     WITH_SPIN_LOCK (&ie->ie_lock) {
-      insert_handler(ie, ih);
+      ie_insert_handler(ie, ih);
```

Comment:
I know it's not really a part of the commit, but this is extremely confusing. Does it even work? I think that removing a handler by using a function that's meant to do the very opposite thing (or at least that's what its name seems to imply) should at least get a comment explaining this mess.

Reply:
As you mentioned, even if this is wrong, I'll not fix it in this PR. Please ask @MichalBlk or @ThomasStachovsky. They already invested some time into _interrupt threads_. Perhaps there's even fix for that in https://github.com/cahirwpz/mimiker/pull/794